wor-paginate.gemspec

```diff
@@ -0,0 +1,33 @@
+$:.push File.expand_path("../lib", __FILE__)
+
+require "wor/paginate/version"
```

Comment:
Hmm are you sure? i tried building the gem without requiring date and it works fine